manual/manual.adoc

```diff
@@ -671,3 +658,32 @@ number of cheats that you can turn to:
 | **IDBEHOLDM** | Gives the tactical survey map.
 | **IDBEHOLDL** | Gives the night vision goggles.
 |==========================
+
+<<<
+
+== Contributing to Freedoom ==
+
+Freedoom is a 
+[free and open source software (FOSS)]https://en.wikipedia.org/wiki/Free_and_open-source_software
+project contributed to by many users around the world. It is available as 
+both free in cost (free as in free beer) and in modification and 
+redistribution rights (free as in free speech) to end users,
+provided that [certain requirenments]https://github.com/freedoom/freedoom/blob/master/COPYING.adoc
+are met.
+
+If you'd like to contribute to the Freedoom project, please check out the 
+[project's page]https://github.com/freedoom/freedoom/ and
+[discussion forum]https://www.doomworld.com/forum/17-freedoom/
```

Comment:
Freedoom now has uses Discord, which is arguable the primary way its developers communicate. It's [here](https://discordapp.com/channels/616148111076884520/616148111076884524). You can probably find instructions elsewhere about it if you want to copy that.

Reply:
That Discord link doesn't seem to be working for me.

Reply:
Added [this](https://discord.gg/9DA3fut) discord link in 798d742